src/layouts/dialog.vue

```diff
@@ -24,6 +47,12 @@ export default Vue.extend({
   html {
     height: initial;
   }
+  :root[data-flex~="width"] #__layout > .wrapper[data-loaded] {
+    width: 100vw;
+  }
+  :root[data-flex~="height"] #__layout > .wrapper[data-loaded] {
+    height: 100vh;
+  }
```

Comment:
This is for the Kubernetes error dialog: it forces the content to be the full height of the window, so that the log view will resize with the window (rather than having blank space if you resize the window larger).